Unexpected panic

```
Compression process has started (keys = 66, threads = 16, time preservation = false). Please be patient...
[1/2] File light.png compressed (63.33 KB → 24.02 KB / -39.31 KB, -62.07%)
[2/2] File dark.png compressed (63.71 KB → 24.07 KB / -39.64 KB, -62.22%)
panic: strings: negative Repeat count

goroutine 1 [running]:
strings.Repeat({0x80c000?, 0x6?}, 0xc0000cd860?)
	strings/strings.go:624 +0x585
gh.tarampamp.am/tinifier/v5/internal/cli.(*fileStats).Table(0xc000186200)
	gh.tarampamp.am/tinifier/v5/internal/cli/stats.go:152 +0x1a8a
gh.tarampamp.am/tinifier/v5/internal/cli.(*App).run(0xc00018c000, {0x8111f0?, 0xc0000d00f0?}, {0xc000012160, 0x2, 0x2})
	gh.tarampamp.am/tinifier/v5/internal/cli/app.go:398 +0x665
gh.tarampamp.am/tinifier/v5/internal/cli.NewApp.func2({0x8111f0, 0xc0000d00f0}, 0x0?, {0xc000012160, 0x2, 0x2})
	gh.tarampamp.am/tinifier/v5/internal/cli/app.go:161 +0x5b2
gh.tarampamp.am/tinifier/v5/internal/cli/cmd.(*Command).Run(0xc00018c000, {0x8111f0?, 0xc0000d00f0?}, {0xc000012160, 0x2, 0x2})
	gh.tarampamp.am/tinifier/v5/internal/cli/cmd/command.go:207 +0x60d
gh.tarampamp.am/tinifier/v5/internal/cli.(*App).Run(...)
	gh.tarampamp.am/tinifier/v5/internal/cli/app.go:194
main.run()
	gh.tarampamp.am/tinifier/v5/cmd/tinifier/main.go:72 +0x1cd
main.main()
	gh.tarampamp.am/tinifier/v5/cmd/tinifier/main.go:19 +0x17
```

--- a/internal/cli/stats.go
+++ b/internal/cli/stats.go
@@ -101,16 +101,16 @@ func (fs *fileStats) Table() string { //nolint:funlen
 		fileName, typeName, diffSize, deltaSize := row[0], row[1], row[2], row[3]
 
 		b.WriteString(fileName)
-		b.WriteString(strings.Repeat(" ", longestFileName-utf8.RuneCountInString(fileName)))
+		b.WriteString(strings.Repeat(" ", max(0, longestFileName-utf8.RuneCountInString(fileName))))
 		b.WriteString(pad)
 
 		b.WriteString(typeName)
-		b.WriteString(strings.Repeat(" ", longestType-utf8.RuneCountInString(typeName)))
+		b.WriteString(strings.Repeat(" ", max(0, longestType-utf8.RuneCountInString(typeName))))
 		b.WriteString(pad)
 
 		if !item.Skipped {
 			b.WriteString(diffSize)
-			b.WriteString(strings.Repeat(" ", longestDiffSize-utf8.RuneCountInString(diffSize)))
+			b.WriteString(strings.Repeat(" ", max(0, longestDiffSize-utf8.RuneCountInString(diffSize))))
 			b.WriteString(pad)
 
 			b.WriteRune('(')
@@ -128,13 +128,13 @@ func (fs *fileStats) Table() string { //nolint:funlen
 	if l := len(fs.Items); l > 1 && totalSkipped < l {
 		b.WriteRune('\n')
 		b.WriteString("   ") // [space][emoji][space]
-		b.WriteString(strings.Repeat(" ", longestFileName))
+		b.WriteString(strings.Repeat(" ", max(0, longestFileName)))
 		b.WriteString(pad)
 
 		const total = "Total:"
 
 		b.WriteString(total)
-		b.WriteString(strings.Repeat(" ", longestType-utf8.RuneCountInString(total)))
+		b.WriteString(strings.Repeat(" ", max(0, longestType-utf8.RuneCountInString(total))))
 		b.WriteString(pad)
 
 		var (
@@ -149,7 +149,7 @@ func (fs *fileStats) Table() string { //nolint:funlen
 		)
 
 		b.WriteString(diffSize)
-		b.WriteString(strings.Repeat(" ", longestDiffSize-utf8.RuneCountInString(diffSize)))
+		b.WriteString(strings.Repeat(" ", max(0, longestDiffSize-utf8.RuneCountInString(diffSize))))
 		b.WriteString(pad)
 
 		b.WriteRune('(')

